### PR TITLE
KML: Parsing of NetworkLink tag

### PR DESCRIPTION
**Pull request by oterral**
This PR add the pasing of NetworkLink tag: https://developers.google.com/kml/documentation/kmlreference#networklink

As in OL2, that made a synchronous http request to load the kml file.

### CODE REVIEW

**Comment by oterral** [comment]
Thx @tsauerwein, I've adapted according to your comments (https://github.com/oterral/ol3/commit/90521caa55dc493e760992c8542c8f6d5e6994a0#commitcomment-9454630)

**Comment by fredj** [comment]
Is a synchronous request really needed?
They are deprecated in Firefox: 
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Synchronous_and_Asynchronous_Requests#Synchronous_request

**Comment by oterral** [comment]
Damn Firefox !!! Ï didn't know that.  May be the other way is to make a readNetworkLinks method (like readName) and let the user handle the links as he wants (using promise or callbacks).

What do you think?

**Comment by oterral** [comment]
@fredj?

**Comment by elemoine** [comment]
@oterral That sounds good to me.

**Comment by fredj** [comment]
LGTM too

**Comment by oterral** [comment]
@fredj @elemoine could you merge this one ?

**Comment by elemoine** [comment]
@oterral I still see the synchronous. I thought your plan was to change this and use a callback (or a promise) for that. Did I misunderstand?

**Comment by fredj** [comment]
> @oterral I still see the synchronous. I thought your plan was to change this and use a callback (or a promise) for that. Did I misunderstand?

I thought the same

**Comment by oterral** [comment]
olalala total misunderstanding sorry.

**Comment by oterral** [comment]
method ` readNetworkLinks `  added

File: src/ol/format/kmlformat.js [hunk 1]
```diff
@@ -1712,6 +1753,71 @@ ol.format.KML.prototype.readNameFromNode = function(node) {
 
 
 /**
+ * @param {Document|Node|string} source Souce.
+ * @return {Array.<Object>} Network links.
+ * @api stable
```
Comment (elemoine) [inline src/ol/format/kmlformat.js]: I don't think we need the stability marker at this point. So please leave `@api` but remove `stable`.

Comment (oterral) [inline src/ol/format/kmlformat.js]: ok I remove

Comment (fredj) [inline src/ol/format/kmlformat.js]: `@api` should be sufficient for now

**Comment by oterral** [comment]
Thank you very much guys!!!

**Comment by bartvde** [comment]
did tests pass for you in Firefox?
I am getting: ```expected 'http://localhost:3000/test/bar/bar.kml' to equal 'bar/bar.kml'```

**Pull request merged**